### DISCUSSION TITLE
license issue

### DISCUSSION THREAD

**Discussion by kimseong-yeon** [bitcraze/discussions#1649] [Q&A]
Hi. i'm using Bolt and additional custom deck for project
if i add additional deck with I2C motordriver board and deck driver code, that source code need gplv3 license ?
i think distribution will be critical issue on my project

**Comment by ataffanel** [comment]
Hi,
Everything running in the STM32 or nRF51 and linked in the Crazyflie firmware is bound by the GPLv3 license. So the easiest if you have code that should not be GPL is to run it on an external MCU.

In your case it means that the deck driver needs to be GPLv3

**Reply by kimseong-yeon** [reply]
thanks for reply,
I wonder is there any options for commercial license?
if available, can you let me know how to contact?

**Reply by Bananazooka** [reply]
Hi, 
For inquiries like that please contact contact@bitcraze.io
